tool/tsh/common/tsh_test.go

```diff
@@ -2379,8 +2378,7 @@ func TestAccessRequestOnLeaf(t *testing.T) {
 // TestSSHCommand tests that a user can access a single SSH node and run commands.
 func TestSSHCommands(t *testing.T) {
 	modules.SetTestModules(t, &modules.TestModules{TestBuildType: modules.BuildEnterprise})
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
+	ctx := t.Context()
```

Comment:
probably not applicable to this instance but we should ensure Cleanup only use context.Background():
https://github.com/gravitational/teleport/blob/de6aa5debf744f948841bf5c7db86742f57d5a91/tool/tsh/common/tsh_test.go#L1749-L1752